packages/protocol/test/libs/LibTrieProof.test.ts

```diff
@@ -62,6 +62,7 @@ describe("integration:LibTrieProof", function () {
             libraries: {
                 LibBridgeProcess: libBridgeProcess.address,
                 LibBridgeRetry: libBridgeRetry.address,
+                LibTrieProof: testLibTreProof.address,
```

Comment:
Is this specifically for testing purposes? I tried running this with libTrieProof.address and it worked still

Reply:
For testing `isMessageFailed`, using testLibTrieProof allows us to modify the destChain's state directly using a new method in TestLibTrieProof.